monocular_pose_estimator_lib/CMakeLists.txt

```diff
@@ -3,21 +3,24 @@ project(monocular_pose_estimator_lib)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -O3")
 
-find_package(catkin REQUIRED COMPONENTS cmake_modules roscpp)
-find_package(Eigen REQUIRED)
+find_package(catkin REQUIRED COMPONENTS roscpp)
 find_package(OpenCV REQUIRED)
```

Comment:
is the library variable called `OpenCV_LIBS` or `OpenCV_LIBRARIES`?

catkin expects the variable to be called `OpenCV_LIBRARIES`. So, it might be necessary to add an alias.